src/20231207_AaveV3Ethereum_GhoIncidentReport_20231126/AaveV3Ethereum_GhoIncidentReport_20231113.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "GHO update on Aave V3 Ethereum Pool for 13/11/2023 Report"
+author: "Aave Labs @aave"
+discussions: "https://governance.aave.com/t/arfc-gho-technical-incident-13-11-2023/15642"
+---
+
+## Simple Summary
+
+This proposal patches the GHO integration with the Aave V3 Pool, fixing an issue reported by Immunefi on November 13, 2023. The patch, developed by Aave Labs in collaboration with Certora, upholds the highest safety standards.
```

Comment:
does it make sense to ad the link to the bug report or it is omitted for purpose?

Reply:
it is omitted in purpose. We ll disclose all the details after AIP execution